### DISCUSSION TITLE
Pause after skipping

### DISCUSSION THREAD

**Discussion by falone** [jellyfin/jellyfin-androidtv#2966] [General]
Hi. I use JF windows server-version 10.8.10 on TiVo Stream 4k.
When i skip video playback is going to pause, so i have to press Play after that.
Is it possible to choose no-pause option?
Thanks in advance.

**Comment by v1talii-dev** [comment]
similar problem but the developers don't plan to implement it in the near future https://github.com/jellyfin/jellyfin-androidtv/issues/4442
